src/XRP/shared/account-root-flags.ts

```diff
@@ -19,6 +19,7 @@ class AccountRootFlags {
   static LSF_REQUIRE_AUTH = 1 << 18 // 0x00040000, 262144
   static LSF_REQUIRE_DEST_TAG = 1 << 17 // 0x00020000, 131072
   static LSF_PASSWORD_SPENT = 1 << 16 // 0x00010000, 65536
+  static NO_FLAG = 0
```

Comment:
cool

Reply:
same note for you as here https://github.com/xpring-eng/Xpring-JS/pull/783#discussion_r502799172